dist/rpm/os-autoinst.spec

```diff
@@ -32,7 +32,7 @@ Source0:        %{name}-%{version}.tar.xz
 %define opencv_require pkgconfig(opencv)
 %endif
 # The following line is generated from dependencies.yaml
-%define build_requires %opencv_require autoconf automake gcc-c++ libtool make perl(ExtUtils::Embed) perl(ExtUtils::MakeMaker) >= 7.12 perl(Module::CPANfile) perl(Pod::Html) pkg-config pkgconfig(fftw3) pkgconfig(libpng) pkgconfig(sndfile) pkgconfig(theoraenc)
+%define build_requires %opencv_require cmake gcc-c++ make perl(Module::CPANfile) perl(Pod::Html) pkg-config pkgconfig(fftw3) pkgconfig(libpng) pkgconfig(sndfile) pkgconfig(theoraenc)
```

Comment:
IMHO we should not even need make here anymore but instead ninja. Then the cmake macros further down automatically use ninja as well

Reply:
I could also use ninja but I haven't noticed any remarkable benefit when testing with it so far. Hence I've just stuck to what works so far. (We can improve this later. As a first step I would be happy if the build would succeed at all and refrain from additional experiments.)

Reply:
Just use ninja, ok? :)

https://build.opensuse.org/package/live_build_log/devel:openQA:TestGithub:OPR-1482/os-autoinst/openSUSE_Tumbleweed/x86_64 shows some completely different problem right now:

```
[   85s] + rpm -q --whatprovides 'perl(Perl::Critic)'
[   85s] no package provides perl(Perl::Critic)
```

Reply:
If you insist. I've already seen the other problems. Any ideas regarding them?

Reply:
I've been switching to ninja. The other problems have been resolved by removing this check which likely doesn't make much sense anymore.